### PR TITLE
Sto32 - UI update 

### PR DESCRIPTION
**Pull request by stephenheap**
In this one:

o The current players turn, remaining actions and remaining turns are shown at the top of the screen.
o Some bug fixes, like removing solutions from chat.
o Better fix for the new turn bug (showing information even when blocked)

### CODE REVIEW

**Comment by TygerBurning** [comment]
111 lines is much more reasonable

File: src/main/java/org/pandemic/game/managers/ActionManager.java [hunk 1]
```diff
@@ -50,6 +50,10 @@ public boolean isActionPossible() {
     private void recalculateActions() {
         availableActions = actionCollector.getAvailableActions(actionsTaken);
     }
+    
+    public int numberOfRemainingActions(){
+        return 4 - currentActions;
```
Comment (TygerBurning) [inline src/main/java/org/pandemic/game/managers/ActionManager.java]: Not relevant right now, but a couple of Events/Roles give extra actions

File: src/main/java/org/pandemic/game/operations/blocker/BlockerPickForecastCards.java [hunk 2]
```diff
@@ -35,12 +35,16 @@ public BlockerPickForecastCards(Player player, Deck<InfectionCard> infectionCard
...
+                    break;
                 case 1:
                     numberString = "2nd";
+                    break;
                 case 2:
                     numberString = "3rd";
+                    break;
                 default:
                     numberString = (i+1) + "th";
+                    break;
```
Comment (TygerBurning) [inline src/main/java/org/pandemic/game/operations/blocker/BlockerPickForecastCards.java]: D'oh. I've done this so many times I should have spotted it. Apologies

File: src/main/webapp/js/drawer.js [hunk 3]
```diff
@@ -215,22 +215,26 @@ function drawMap() {
...
-            
-            '<div id = "game-message">',
-            '<input id = "game-message-content" type="text" name="message" placeholder="Type here..." onfocus="this.placeholder = \'\'" onblur="this.placeholder = \'Type here...\'">',
-            '<div id="undo" onclick="send_undo_response()">Undo</div>',
-            '<div id="end-turn" onclick="send_end_turn_response()">End Turn</div>',
-            '<div id = "game-message-submit">Submit</div>',
-            '</div>',
-            
+                '<div id = "status-section">',
+                    '<span id = "remaining-actions">3 Actions remaining</span>',
```
Comment (TygerBurning) [inline src/main/webapp/js/drawer.js]: 3 actions?

**Comment by TygerBurning** [comment]
LGTM

File: src/main/webapp/js/drawer.js (hunk 3 above)
Comment (stephenheap) [inline src/main/webapp/js/drawer.js]: Whoops, I originally hardcoded when making the css work, and forgot to take this out :P It didn't do anything anyway, as it was instantly overwritten, but it's gone now :p

File: src/main/java/org/pandemic/game/managers/ActionManager.java (hunk 1 above)
Comment (stephenheap) [inline src/main/java/org/pandemic/game/managers/ActionManager.java]: I've added a variable `currentMaxActions` which is hardcoded to 4 for now, but can be changed for these Roles/Events :)

**Pull request merged**